com.unity.netcode.gameobjects/Editor/NetworkObjectEditor.cs

```diff
@@ -114,10 +114,34 @@ public override void OnInspectorGUI()
             {
                 EditorGUI.BeginChangeCheck();
                 serializedObject.UpdateIfRequiredOrScript();
+                var ownershipProperty = serializedObject.FindProperty(nameof(NetworkObject.Ownership));
+                var sceneObjectProperty = serializedObject.FindProperty(nameof(NetworkObject.IsSceneObject));
+                var previousOwnership = (NetworkObject.OwnershipStatus)ownershipProperty.intValue;
+                var allFilter = NetworkObject.OwnershipStatus.RequestRequired | NetworkObject.OwnershipStatus.Transferable | NetworkObject.OwnershipStatus.Distributable;
```

Comment:
Is there any way this can be saved somewhere so it doesn't need to be recalculated each time?